src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs

```diff
@@ -50,7 +51,12 @@ public OpenApiDocument Read(TextReader input, out OpenApiDiagnostic diagnostic)
                 return new OpenApiDocument();
             }
 
-            return new OpenApiYamlDocumentReader(this._settings).Read(yamlDocument, out diagnostic);
+            var asJsonNode = yamlDocument.ToJsonNode();
```

Comment:
I believe we are going to make the switch to do Yaml => JsonNodes => OpenAPIDocument.  This will facilitate us to use the JsonSchema.Net Examples and hopefully get rid of OpenApiAny and all of the OpenApiPrimitive classes.